### PR TITLE
Build 0.4.1

### PR DESCRIPTION
**Pull request by akabanovs**
- version and sha256 updated.
     - noarch removed.
     - build script updated.
     - host dependencies updated.
     - run dependencies updated.
     - test section updated.
     - 'about' section updated.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -8,53 +8,68 @@ package:
...
-    - triad
+    - antlr4-python3-runtime >=4.11.1,<4.12
+    - pandas >=1.2.0
+    - python
+    - triad >=0.8.0
 
 test:
-  source_files:
-    - tests
+  #source_files:
```
Comment (M-Waszkiewicz-Anaconda) [inline recipe/meta.yaml]: is this needed ?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -8,53 +8,68 @@ package:
...
+    # modin-ray and ray-default: no support for python 3.11 and missing on all archs except for linux-64 and win-64  
+    - qpd_ray  # [py<311 and (linux64 or win)]
     - qpd_test
     - qpd_version
   commands:
     - pip check
-    # Comment out test suite until ray issue resolved
-#     - pytest -vvv --capture=tee-sys tests
+    # Comment out test suite until ray issue is resolved
+    #- pytest -vvv --capture=tee-sys tests
```
Comment (M-Waszkiewicz-Anaconda) [inline recipe/meta.yaml]: can we also add a selector here  # [py<311 and (linux64 or win)] ?

File: recipe/meta.yaml (hunk 1 above)
Comment (akabanovs) [reply recipe/meta.yaml]: This is commented out, so when we sort out constraints with ray package, we could uncomment and switch on tests.

File: recipe/meta.yaml (hunk 2 above)
Comment (akabanovs) [reply recipe/meta.yaml]: Same as above, this is commented out. Tests require ray. And I was still having problems with tests when running on linux-64 and win-64 with py<311. CF has this commented out, so I did the same...

**Review by ELundby45 — APPROVED** [review]
LGTM

**Review by ELundby45 — APPROVED** [review]

**Review by M-Waszkiewicz-Anaconda — COMMENTED** [review]
LGTM

**Review by M-Waszkiewicz-Anaconda — APPROVED** [review]
LGTM

**Pull request merged**